src/elements/content-sidebar/versions/VersionsSidebar.js

```diff
@@ -49,8 +61,26 @@ const VersionsSidebar = ({ error, isLoading, parentName, versions, ...rest }: Pr
         >
             <LoadingIndicatorWrapper className="bcs-Versions-content" crawlerPosition="top" isLoading={isLoading}>
                 {showError && (
-                    <InlineError title={<FormattedMessage {...messages.versionServerError} />}>
+                    <InlineError
+                        title={
+                            errorUpsell ? (
+                                <FormattedMessage {...messages.versionServerErrorUpsell} />
+                            ) : (
+                                <FormattedMessage {...messages.versionServerError} />
+                            )
+                        }
+                    >
                         <FormattedMessage {...error} />
+                        {errorUpsell && (
+                            <PlainButton
+                                onClick={onUpgradeCTAClick}
+                                data-resin-target="jessica-to-update"
+                                type="button"
+                                className="upgradeNowButton"
```

Comment:
Please use SUIT style naming conventions to stay consistent with the other style rules. Maybe `bcs-Versions-upgrade`?